projects/views.py

```diff
@@ -17,20 +17,18 @@ def post_detail_view(request):
 
 
 def room86(request):
+    import random
     print(request)
     posts = Post.objects.all()
-    featured_post = random_posts(posts)
+    featured_post = random.choice(posts)
     template = "room86.html"
     context = {
-        'myposts': featured_post
+        'myposts': posts,
```

Comment:
myposts and randompost is the varible name that we are going to use to render those data in the template